lock-api/src/main/java/com/palantir/lock/client/LockLeaseService.java

```diff
@@ -65,8 +79,22 @@ LockImmutableTimestampResponse lockImmutableTimestamp() {
     }
 
     StartTransactionResponseV4 startTransactions(int batchSize) {
-        StartTransactionRequestV4 request = StartTransactionRequestV4.createForRequestor(clientId, batchSize);
-        StartTransactionResponseV4 response = delegate.startTransactions(request);
+        final StartTransactionResponseV4 response;
+        if (HAVE_ROLLED_OUT_CONJURE_CHANGES_INTERNALLY) {
+            ConjureStartTransactionsRequest request = ConjureStartTransactionsRequest.builder()
```

Comment:
James: I did this because conjure objects are naturally backwards compatible, whereas our objects typically are not (or at least, require the user to have correctly added jsonignoreproperties) - it's safe to add new fields on the server side. This means we should be able to evolve our types rather than adding new endpoints for every field.